packages/documentation/src/pages/getting-started/common-tasks/new-unit-test.mdx

```diff
@@ -0,0 +1,141 @@
+---
+title: Writing a unit test
+tags: Mocha, Chai, Enzyme, schemaform-utils,
+---
+
+# Writing a unit test
+
+## Unit testing overview
+
+- `vets-website` uses
+  - [Mocha](https://mochajs.org/) for running unit tests
+  - [Chai](http://chaijs.com/) for test assertions
+  - [Enzyme](https://airbnb.io/enzyme/) for mounting and inspecting React components
+- Unit tests are **collocated with application folder** in a test directory that matches the application directory structure
+```
+- 📂 MyApplication
+  - 📂 components
+    - MyComponent.jsx
+  - 📂 tests
+    - 📂 components
+      - MyComponent.unit.spec.jsx
+```
+- Mocha runs all unit tests under `/src` folder that end with `.unit.spec.js(x)`
+
+## Unit test conventions
+```
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+import MyComponent from '../../components/MyComponent';
+
+describe('MyApplication', () => {
+  describe('MyComponent', () => {
+    it('renders privacy act disclosure when state.show is true', () => {
+        const componentUnderTest = shallow(<MyComponent />);
```

Comment:
open to other conventions and other ways to present. I assume that everyone who reads this has written a unit test so just showing an example with most of our conventions is a quick way to convey them.